ui/components/multichain/toast/toast.tsx

```diff
@@ -66,18 +82,49 @@ export const Toast = ({
     <BannerBase
       data-theme={theme === ThemeType.light ? ThemeType.dark : ThemeType.light}
       onClose={onClose}
-      borderRadius={borderRadius}
+      backgroundColor={BackgroundColor.backgroundSection}
+      borderWidth={1}
+      borderColor={BorderColor.borderMuted}
+      borderRadius={borderRadius || BorderRadius.XL}
       data-testid={dataTestId ? `${dataTestId}-banner-base` : undefined}
       className={`toasts-container__banner-base ${className}`}
     >
-      <Box display={Display.Flex} gap={4} data-testid={dataTestId}>
+      <Box
+        display={Display.Flex}
+        gap={3}
+        data-testid={dataTestId}
+        {...contentProps}
+      >
         {startAdornment}
         <Box>
-          <Text className="toast-text" variant={textVariant}>
+          <Text
+            className="toast-text"
+            variant={textVariant || TextVariant.bodyMdMedium}
+          >
             {text}
           </Text>
+          {description && (
+            <Text
+              className="toast-text"
+              variant={descriptionVariant || TextVariant.bodySm}
+              color={TextColor.textAlternative}
+            >
+              {description}
+            </Text>
+          )}
           {actionText && onActionClick ? (
-            <ButtonLink onClick={onActionClick}>{actionText}</ButtonLink>
+            <Button
+              variant={ButtonVariant.Secondary}
+              size={ButtonSize.Sm}
+              textProps={{
+                fontWeight: FontWeight.Regular,
+                variant: DsTextVariant.BodySm,
```

Comment:
```suggestion
                variant: DsTextVariant.BodySmMedium,
```
Also recommend removing `fontWeight: FontWeight.Regular`. This is so that all CTAs have the same weights.